content/docs/api-reference/blockchain-api/index.md

```diff
@@ -17,7 +17,7 @@ toc: true
 
 Create the [bearer_token](../../setup-and-administration/getting-access-tokens-using-app-registrations) and store in a file in a secure local directory with 0600 permissions.
 
-### Fetch Transactions for an event (v1alpha1)
+### Fetch Transactions for an event (v1alpha2)
```

Comment:
We now return different shapes based on whether the event has a proof mechanism of `simplehash` or `khipu`, we should surface these changes i feel?

If the event is `simplehash`, it will return a list that contains ONLY 1 transaction, this is the anchor transaction and will contain the rawTransaction as well as `simple_hash_details` with information on how to reproduce the anchor.

If the event is `khipu`, it will return a list of 1 or more transactions, these will contain the rawTransaction, and `khipu_details`, but `khipu_details` is empty for now.

We added an extra field to each Transaction, called `kind` which describes which `proofMechanism` it is (simple hash or khipu), so you can tell what kind of details you should be looking at.

Shout if this needs more context :)

Reply:
I updated the example response and will add additional information in the future, as discussed.